docs/source/notebooks/opendap.ipynb

```diff
@@ -26,8 +26,9 @@
    "source": [
     "%matplotlib inline\n",
     "import warnings\n",
-    "import numpy as np\n",
+    "\n",
     "import netCDF4 as nc\n",
+    "import numpy as np\n",
```

Comment:
This auto import re-ordering might possibly be problematic in some specific case.  Is there any case where we need some specific import ordering?  Ex: some module monkeypatch another module.

Reply:
Valid concern. There used to be an issue with the warning that were raised from CLISOPS on import, but that was dealt with a while ago. xarray-based accessors aren't load-order dependent, so I don't imagine there'd be an issue there either.